client/containers/Booking/Booking.jsx

```diff
@@ -61,44 +58,42 @@ export class Booking extends Component {
                 {
                     this.props.appointment.isSubmitted ?
                         <div className="requestDiv">
-                            <p className="requestText">Your Request has been submitted successfully for Approval</p>
+                                    <p className="requestText">Your request is under reviewed, you will receive an email once your request is reviewed</p>
```

Comment:
Your request is under review.
You will receive an email once your request is reviewed.